lms/djangoapps/ccx/views.py

```diff
@@ -224,6 +224,7 @@ def override_fields(parent, data, graded, earliest=None, ccx_ids_to_delete=None)
                 ccx_ids_to_delete.append(get_override_for_ccx(ccx, block, 'start_id'))
                 clear_ccx_field_info_from_ccx_map(ccx, block, 'start')
 
+            # Only subsection (aka sequential) has due date
             due = parse_date(unit['due'])
             if due:
```

Comment:
since `parse_date` can return `None`, this should be `if due is not None`, unless it is making other kind of checks.

Anyway, since you did not change this code, feel free to leave it as is